src/content/docs/es/guides/view-transitions.mdx

```diff
@@ -215,37 +215,70 @@ export interface TransitionDirectionalAnimations {
 }
 ```
 
-El siguiente ejemplo muestra todas las propiedades necesarias para definir una animación personalizada de `fade`:
+El siguiente ejemplo muestra todas las propiedades necesarias para definir una animación personalizada de `bump (impacto/rebote) dentro de una etiqueta `<style is:global> en el archivo de diseño (layout) raíz:
```

Comment:
```suggestion
El siguiente ejemplo muestra todas las propiedades necesarias para definir una animación personalizada de `bump (impacto/rebote) dentro de una etiqueta `<style is:global>` en el archivo de diseño (layout) raíz:
```
I think this will fix the build issue!